gk/main.c

```diff
@@ -2442,6 +2443,8 @@ gk_stage1(void *arg)
 		}
 		inst_ptr->tx_queue_back = ret;
 
+		inst_ptr->sol_inst = &sol_conf->instances[gk_conf->gk_sol_map[i]];
```

Comment:
The code should verify that `gk_conf->gk_sol_map[i]` is a valid index of `sol_conf->instances[]` (i.e. that it is greater or equal to zero and less than the number of SOL instances. We need to test it here because the Lua code could have a bug and pass an invalid map.

Reply:
When comparing ``gk_conf->gk_sol_map[i] >= 0``, an error will happen ``error: comparison of unsigned expression >= 0 is always true``.

Reply:
Given that `gk_conf->gk_sol_map[i]` is an `unsigned int`, it's okay to skip the `>= 0` test.

Reply:
Don't use `RTE_VERIFY()` here, have an `if` statement and deal as a regular error.